algoliasearch-common/src/main/java/com/algolia/search/objects/Query.java

```diff
@@ -289,15 +297,27 @@ public String toParam() {
     return builder.toString();
   }
 
-  public Query setAroundRadius(Integer aroundRadius) {
+  public Query setAroundRadius(AroundRadius aroundRadius) {
     this.aroundRadius = aroundRadius;
     return this;
   }
 
-  @JsonIgnore
+  /**
+   * @deprecated use setAroundRadius(AroundRadius)
+   * @return
+   */
+  @Deprecated()
+  public Query setAroundRadius(Integer aroundRadius) {
```

Comment:
Apparently, the tests are failing because of a conflict in the setters for `AroundRadius`.
One of these two methods should be annotated with `@JsonIgnore`.